aws/resource_aws_sagemaker_domain_test.go

```diff
@@ -3,24 +3,72 @@ package aws
 import (
 	"fmt"
 	"log"
+	"os"
 	"regexp"
 	"testing"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/sagemaker"
+	"github.com/hashicorp/go-multierror"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 	"github.com/terraform-providers/terraform-provider-aws/aws/internal/service/sagemaker/finder"
 )
 
+func TestAccAWSSagemakerDomain_serial(t *testing.T) {
```

Comment:
Maybe add a comment noting that SageMaker app and user profile tests are also serialized as they reference a SageMaker domain resource and there is a low overall account limit to the number of domains.